verify/cockpit-verify

```diff
@@ -22,12 +22,19 @@ mkdir -p mock
 
 echo "Starting testing"
 
+iteration=0
 while true; do
     fail=0
     find cockpit -name '*.py?' -delete
     git -C cockpit fetch origin
     git -C cockpit reset --hard origin/master
-    cockpit/test/vm-download --prune
+    let prune=iteration%25
+    if [ "$prune" -eq 0 ]; then
+        cockpit/test/vm-download --prune
+    else
+        cockpit/test/vm-download
+    fi
+    let "iteration++"
```

Comment:
... do `iteration=$(( (iteration+1) % 25 ))` here instead, which is POSIX shell.